StRoot/StTriggerUtilities/Bemc/StBemcTriggerSimu.cxx

```diff
@@ -1071,7 +1071,10 @@ void StBemcTriggerSimu::FEEout2009()
   // emcSim->setCheckStatus(kBarrelEmcTowerId,false);
   // emcSim->setMakeFullDetector(kBarrelEmcTowerId,true);
   // emcSim->setDoZeroSuppression(kBarrelEmcTowerId,false);
-  assert(nhits == kNTowers);
```

Comment:
Zilong, thanks. Indeed, I was worried about this change. The reason was that the code crashes at this line, see the log file
/star/u/dongx/pwg/Git/20220224/test/test.log
or under that directory, run
"bfc.C(10,"pp2022,StiCA,BEmcChkStat,epdhit,-hitfilt,-evout,-fcsCluster,-fcsPoint","/gpfs01/star/daq/2022/032/23032031/st_physics_adc_23032031_raw_6500079.daq")"
As you mentioned, this may be related to the pedestal DB entry that is related to what Zhenyu is trying to fix. If you get a chance, maybe you have a better solution. Thanks